src/patchelf.cc

```diff
@@ -2242,6 +2493,25 @@ int mainWrapped(int argc, char * * argv)
         else if (arg == "--add-debug-tag") {
             addDebugTag = true;
         }
+        else if (arg == "--rename-dynamic-symbols") {
+            renameDynamicSymbols = true;
+            if (++i == argc) error("missing argument");
+
+            std::ifstream infile(argv[i]);
+            if (!infile) error(fmt("Cannot open map file ", argv[i]));
+
+            std::string from, to;
+            while (true)
+            {
+                if (!(infile >> from))
+                    break;
+                if (!(infile >> to))
+                    error("Odd number of symbols in map file");
+                if (symbolsToRenameKeys.count(from))
+                    error(fmt("Symbol appears twice in the map file: ", from.c_str()));
```

Comment:
Can we keep it c++17 for now? https://en.cppreference.com/w/cpp/utility/format/format
patchelf gets really used in many odd corners (i.e. for bootstrapping) you wouldn't expect and people were already mad at me to that they have to now install a c++17 ready compiler...

Reply:
I suppose you're talking about the `fmt` call. That's a function implemented in `patchelf.cc` some lines above, it's not the C++ format.